scripts/jest/index.js

```diff
@@ -1,4 +1,8 @@
-module.exports = {
+// @ts-check
+const lernaAlias = require('../lernaAliasNorthstar');
```

Comment:
should we rename this to `northStar.js` or something to indicate more clearly this is used for northstar only?

Reply:
Maybe, but we have this problem in various config files, so I don't consider it as important to fix right now. (though since I'm already changing all the configs using it, this might be a decent time)

Reply:
ok; don't worry about it now